changelogs/1188-migrate-module_utils-backup.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
```

Comment:
i might categorize this as `trivial` since no changes were made from the perspective of the end-user

Reply:
I agree, updated this, actually the file was in the wrong dir.